src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -963,11 +963,12 @@ private InstructorFeedbackResultsQuestionTable buildQuestionTableAndResponseRows
             }
             
             // If question specific sorting is not needed, responses are sorted
-            // by default order (first by team name, then by display name)
+            // in the following order (first by recipient name, then by team name,
+            // then by display name)
             if (questionDetails.isQuestionSpecificSortingRequired()) {
                 Collections.sort(responseRows, questionDetails.getResponseRowsSortOrder());
             } else {
-                responseRows = InstructorFeedbackResultsResponseRow.sortListWithDefaultOrder(responseRows);
+                responseRows = InstructorFeedbackResultsResponseRow.sortListByRecipientNameThenDefaultOrder(responseRows);
```

Comment:
Yes, constant sum questions are the questions where the giver distribute points among options. 

This issue refers to changing the display of the statistics by default.